src/javascript/SelectorTypes/registerSelectorTypesOnChange.js

```diff
@@ -82,7 +87,7 @@ export const registerSelectorTypesOnChange = registry => {
                     }
                 });
                 if (updated) {
-                    onChangeContext.onSectionsUpdate();
+                    onChangeContext.setSections(sections);
```

Comment:
This will log the warning message:
`console.warn('Sections update is deprecated');` may be better to keep calling `onSectionsUpdate()` to avoid warn logs.